### PR TITLE
ingest: use new augur curate commands

### PR DESCRIPTION
**Pull request by j23414**
## Description of proposed changes

Updates the curate rule to use the new augur curate commands.
Updates ingest/vendored to pull in latest changes including https://github.com/nextstrain/ingest/pull/45.

Includes additional changes to rename `transform` to `curate` to conform the [pathogen-repo-guide](https://github.com/nextstrain/pathogen-repo-guide/blob/89b3c5dbc9b8f6a009f4a19c3ac56113bc5511ee/ingest/rules/curate.smk#L57)
Includes additional change to remove `reverse` and `institution` columns since they are blank and seemingly unused.

## Related issue(s)

Mirroring changes in https://github.com/nextstrain/pathogen-repo-guide/pull/53
Resolves https://github.com/nextstrain/rsv/issues/67

## Checklist

- [x] Checks pass
- [x] Confirmed only diff is the added period to `<first author> et al.` and the removal of the `reverse` and `institution` columns.

### CODE REVIEW

**Review by genehack — APPROVED** [review]
Tiny nit: if you use `git mv` to rename files, rather than delete/re-add, I think there's better history and change tracking.

**Comment by j23414** [comment]
Ohh interesting, I did use `git mv` in https://github.com/nextstrain/rsv/pull/70/commits/5690058759d693517833af6d9e5c854190164ee8 but perhaps the subsequent changes muddied the history.

**Comment by tsibley** [comment]
> Tiny nit: if you use `git mv` to rename files, rather than delete/re-add, I think there's better history and change tracking.

`git mv` is exactly the same operation as `rm` + `add`. Tracking of moves/copies is done entirely by content similarity. There are config options to tweak how "hard" git tries to identify moves/copies; they're pretty conservative by default and it's nice to tune them upwards sometimes. I've also noticed that github.com's diff sometimes has worse move/copy detection than what I have configured locally.

**Comment by genehack** [comment]
> `git mv` is exactly the same operation as `rm` + `add`. 

TIL!

File: ingest/config/config.yaml [hunk 1]
```diff
@@ -70,7 +70,5 @@ transform:
     'date_submitted',
     'sra_accession',
     'abbr_authors',
-    'reverse',
     'authors',
-    'institution'
```
Comment (joverlee521) [inline ingest/config/config.yaml:73]: _non-blocking_

FWIW, `institution` can be populated by adding `submitter-affiliation=institution` to the `field_map` config.

**Review by joverlee521 — APPROVED** [review]

File: ingest/config/config.yaml (hunk 1 above)
Comment (j23414) [reply ingest/config/config.yaml:73]: Thanks! Looking around the code configs, I decided to add a functionally populated `institution` column in https://github.com/nextstrain/rsv/pull/70/commits/644faa9e3e910f4e4a63c7de04952674a081a2d1

File: ingest/config/config.yaml [hunk 2]
```diff
@@ -8,7 +8,7 @@ ncbi_taxon_id:
   general: "11250"
 
 # Params for the transform rulegeneral
```
Comment (genehack) [inline ingest/config/config.yaml:10]: probably want to update the comment too?

**Review by genehack — APPROVED** [review]

File: ingest/config/config.yaml (hunk 2 above)
Comment (j23414) [reply ingest/config/config.yaml:10]: lol, good catch, will update

**Review by genehack — APPROVED** [review]

**Pull request merged**